x-pack/osquerybeat/beater/config_plugin.go

```diff
@@ -18,9 +17,9 @@ import (
 )
 
 const (
-	configName           = "osq_config"
-	scheduleSplayPercent = 10
-	maxECSMappingDepth   = 25 // Max ECS dot delimited key path, that is sufficient for the current ECS mapping
+	configName                  = "osq_config"
+	defaultScheduleSplayPercent = 10
```

Comment:
What does this do?

Reply:
Value in percent from 1-100 to splay the queries execution interval:

https://github.com/osquery/osquery/blob/2cd5b42c8f8fa52c6d251c6537595c7c59c90f4c/tools/deployment/osquery.example.conf#L22

https://github.com/osquery/osquery/blob/d2be385d71f401c85872f00d479df8f499164c5a/osquery/config/packs.h#L162

It's set to percent 1-100 .... lots of places in oquery source code and kolide launcher set it to 10by default

Reply:
Thanks!